### PR TITLE
CW - SP Migration changed to Product Upgrade

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Change in phrasing needs to be reflected in the file.

# Target branches

Which documentation version does this PR apply to?
- [x] master (Default)
- [ ] manager-4.3

# Links
- Related development PR #<insert PR link, if any>
- Fixes https://github.com/SUSE/spacewalk/issues/22582

### CODE REVIEW

File: modules/common-workflows/pages/workflow-product-migration-webui.adoc [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
-. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP5 x86_64"
+. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for [literal]``SUSE Linux Enterprise Server 15 SP5 x86_64``.
 . Select the recommended channels.
 +
 image::upgradesles01.png[scaledwidth=80%]
 . Click btn:[Add Products].
 +
 image::upgradesles02.png[scaledwidth=80%]
-. Navigate to menu:Systems[registered client > Software > SP Migration].
+. Navigate to menu:Systems[Registered client > Software > SP Migration].
```
Comment (keichwa) [inline modules/common-workflows/pages/workflow-product-migration-webui.adoc:26]:
```suggestion
. Navigate to menu:Systems[Registered client > Software > Product Migration].
```

Comment (keichwa) [reply modules/common-workflows/pages/workflow-product-migration-webui.adoc:26]: TBD at a sync meeting: Do we really want to call this `Systems[Registered client]`?  I think initially we called this the `System Details` page (or tab).

File: modules/common-workflows/pages/workflow-product-migration-webui.adoc [hunk 2]
```diff
@@ -62,6 +58,6 @@ ANSI_COLOR="0;32"
 CPE_NAME="cpe:/o:suse:sles:15:sp5"
 DOCUMENTATION_URL="https://documentation.suse.com/"
 ----
-. On the {productname} {webui} side, you can verify the succesfully completed migration by going to menu:Systems[registered client > Detail].
+. On the {productname} {webui} side, you can verify the succesfully completed migration by going to menu:Systems[Registered client > Detail].
```
Comment (keichwa) [inline modules/common-workflows/pages/workflow-product-migration-webui.adoc:61]:
```suggestion
. On the {productname} {webui} side, you can verify the successfully completed migration by going to menu:Systems[Registered client > Details].
```

**Review by keichwa — CHANGES_REQUESTED** [review]
It would be good if you could also update the PNGs.

**Comment by 0rnela** [comment]
@keichwa -- I have created a separate card for the images update. https://github.com/SUSE/spacewalk/issues/22607

This PR is ONLY about the basic wording changes. B

File: modules/common-workflows/pages/workflow-product-migration-webui.adoc (hunk 1 above)
Comment (0rnela) [reply modules/common-workflows/pages/workflow-product-migration-webui.adoc]: Added to the Confluence.

File: modules/common-workflows/pages/workflow-product-upgrade-webui.adoc [hunk 3]
```diff
@@ -1,33 +1,29 @@
...
-. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for "SUSE Linux Enterprise Server 15 SP5 x86_64"
+. Log in to {productname} {webui} and navigate to menu:Admin[Products] and search for [literal]``SUSE Linux Enterprise Server 15 SP5 x86_64``.
 . Select the recommended channels.
 +
 image::upgradesles01.png[scaledwidth=80%]
 . Click btn:[Add Products].
 +
 image::upgradesles02.png[scaledwidth=80%]
-. Navigate to menu:Systems[registered client > Software > SP Migration].
+. Navigate to menu:Systems[Registered client > Software > SP Migration].
```
Comment (keichwa) [inline modules/common-workflows/pages/workflow-product-upgrade-webui.adoc:26]:
```suggestion
. Navigate to menu:Systems[Registered client > Software > Product Migration].
```

File: modules/common-workflows/pages/workflow-product-upgrade-webui.adoc [hunk 4]
```diff
@@ -62,6 +58,6 @@ ANSI_COLOR="0;32"
 CPE_NAME="cpe:/o:suse:sles:15:sp5"
 DOCUMENTATION_URL="https://documentation.suse.com/"
 ----
-. On the {productname} {webui} side, you can verify the succesfully completed migration by going to menu:Systems[registered client > Detail].
+. On the {productname} {webui} side, you can verify the succesfully completed product upgrade by going to menu:Systems[Registered client > Detail].
```
Comment (keichwa) [inline modules/common-workflows/pages/workflow-product-upgrade-webui.adoc:61]:
```suggestion
. On the {productname} {webui} side, you can verify the succesfully completed product upgrade by going to menu:Systems[Registered client > Details].
```

**Review by keichwa — CHANGES_REQUESTED** [review]
I think there are still 2 small wording adjustments open.  Can you please check the suggestions again?

**Review by keichwa — APPROVED** [review]

**Pull request merged**